packages/three-vrm-animation/package.json

```diff
@@ -20,6 +20,14 @@
       ]
     }
   },
+  "exports": {
+    ".": {
+      "types": "./types/index.d.ts",
```

Comment:
[publint](https://github.com/bluwy/publint) says

```
1. pkg.exports["."].types types is interpreted as ESM when resolving with the "require" condition. 
This causes the types to only work when dynamically importing the package, 
even though the package exports CJS. 
Consider splitting out two "types" conditions for "import" and "require", 
and use the .cts extension, e.g. pkg.exports["."].require.types: "./types/index.d.cts"
```

but seperating declarations to `mts` and `cts` does not make many sense to me and cause trouble for building.